common/src/chain/config/regtest.rs

```diff
@@ -0,0 +1,430 @@
+// Copyright (c) 2022 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::{
+    collections::HashMap,
+    fmt::{Display, Formatter},
+    str::FromStr,
+};
+
+use crate::{
+    chain::{
+        block::timestamp::BlockTimestamp,
+        config::{StakePoolData, DEFAULT_INITIAL_MINT, MIN_STAKE_POOL_PLEDGE},
+        output_value::OutputValue,
+        transaction::Destination,
+        Genesis, PoolId, TxOutput,
+    },
+    primitives::{per_thousand::PerThousand, Amount},
+};
+use crypto::{
+    key::{PrivateKey, PublicKey},
+    vrf::{VRFPrivateKey, VRFPublicKey},
+};
+use hex::FromHex;
+use serialization::hex::HexEncode;
+
+const GENESIS_POOL_ID: &str = "123c4c600097c513e088b9be62069f0c74c7671c523c8e3469a1c3f14b7ea2c4";
+const GENESIS_STAKE_PRIVATE_KEY: &str =
+    "008717e6946febd3a33ccdc3f3a27629ec80c33461c33a0fc56b4836fcedd26638";
+const GENESIS_VRF_PRIVATE_KEY: &str = "003fcf7b813bec2a293f574b842988895278b396dd72471de2583b242097a59f06e9f3cd7b78d45750afd17292031373fddb5e7a8090db51221038f5e05f29998e";
```

Comment:
Could these be proper types, like `PoolId`, `PrivateKey`, `VRFPrivateKey` instead of strings? You can use the `hex!` macro from the `hex` crate to enter hex strings and get a `[u8; N]`.

Reply:
I think it's best to keep them as hex strings, because this is the same format as what is set on the command line, and also to be consistent with the other networks (defined in `mod.rs`).